### ISSUE TITLE
ActiveMQ Localhost connection

### ISSUE THREAD

**Issue by berkerdemirer** [rumblex/angularattack2016-bjaanes#4]
Hey there,

I am trying to read the topics coming from my localhost but i could not done it yet. I do not know where the problem is, i am entering my localhost adress to host field, port is 1883, but in topic field im not sure what to write i mean i do now know the exact path of it in active mq. Can you help me with that?

**Comment by gjermundgaraba** [comment]
Are you running everything (app, mqtt-bridge) on your local enviroment? Or are you using the demo hosted here http://bjaanes.2016.angularattack.io/ ?

You need to run it locally. I have used it with localhost previously. Can you also try with something like 127.0.0.1 to see if that helps?

**Comment by berkerdemirer** [comment]
Yes, i mean i started it locally, i cant still reach to your topics over mosquito but i cant find a path to my topics over activemq
